{{cookiecutter.project_slug}}/{{cookiecutter.package_name}}/client.py

```diff
@@ -1,48 +1,83 @@
-import os
 import logging
+import json
+from typing import Mapping, Dict, Literal
```

Comment:
you don't need `Dict` anymore starting from python 3.9. This is deprecated in favor of directly using `dict`

Reply:
thanks, I'll remove them